_health-care/_js/_components/address-state.jsx

```diff
@@ -0,0 +1,112 @@
+import React from 'react';
+import _ from 'lodash';
+
+class AddressState extends React.Component {
+  constructor() {
+    super();
+    this.state = { hasError: false };
+    this.handleChange = this.handleChange.bind(this);
+  }
+
+  componentWillMount() {
+    this.id = _.uniqueId();
+  }
+
+  handleChange() {
+    const state = this.refs.StateInput.value;
+
+    if (this.validate(state)) {
+      this.setState({ hasError: false });
+    } else {
+      this.setState({ hasError: true });
+    }
+
+    this.props.onUserInput(state);
+  }
+
+  validate(field) {
+    if (field === '') return true;
```

Comment:
This is not the right syntax. It should be: 
```
if (field === '') {
    return true;
}
```
Also, we're returning true either way. Is this a required field? If not, I don't think we even need to validate at all. If you want to keep the validation logic in here once we find out more information about what fields are required or not, I would just include `return true;` for now without checking `if (field === '')`.